CMakeLists.txt

```diff
@@ -76,3 +80,34 @@ if (RC_ENABLE_EXAMPLES)
 endif()
 
 add_subdirectory(extras)
+
+# install the targets and include directories
+install(TARGETS rapidcheck
+	ARCHIVE DESTINATION "${CMAKE_INSTALL_PREFIX}/lib"
+	)
+
+install(DIRECTORY "include/"
+	DESTINATION "${CMAKE_INSTALL_PREFIX}/include"
```

Comment:
According to the docs, relative paths are interpreted as being relative to  `CMAKE_INSTALL_PREFIX` anyway so I would suggest simply specifying this as `DESTINATION "include"`.